zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/NoopExporterConfiguration.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.exporter;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public final class NoopExporterConfiguration {
+  public final ElasticsearchConfig elasticsearch = new ElasticsearchConfig();
+
+  public static final class ElasticsearchConfig {
+    public String url;
+    public IndexSpecificSettings defaultSettings = new IndexSpecificSettings();
+    public Map<String, String> replicasByIndexName = new HashMap<>();
+    public Map<String, String> shardsByIndexName = new HashMap<>();
+  }
+
+  public static final class IndexSpecificSettings {
+    public String numberOfShards = "1";
```

Comment:
we expect these to be passed as integers from the configuration, exposing them as strings might lead to miss-use.